bullet_train/app/views/account/users/edit.html.erb

```diff
@@ -23,18 +23,35 @@
         <%= render 'account/users/oauth' %>
 
         <%= render 'account/shared/box', divider: true do |box| %>
-          <% box.title t('.password.header') %>
+          <% box.title t('.email_and_password.header') %>
+          <% box.description t('.email_and_password.description') %>
```

Comment:
Apologies for being really nitpicky, but could this be `email_or_password`? We can technically edit one or the other, so I feel like matching these keys with the other additions in this PR is appropriate. I'm open if this is commonly done in other web apps though.

Reply:
@gazayas I originally went with `email_or_password` but then realized that it reads weird on the page if the header says "Update Your Email Address or Password" when you can actually change both with the same form. I finally decided that for purposes of displaying them it should be "and" mostly because they live in the same form and so the form is for "email _and_ password". (Technically it's "email and/or password" but I don't think we want to be _that_ precise.)